crates/bevy_asset/src/asset_server.rs

```diff
@@ -558,6 +558,41 @@ impl AssetServer {
             }
         }
     }
+    /// Loads an asset and blocks until it either loads of fails to load. Useful for loading
+    /// default assets and for when startup times are a non issue such as when prototyping.
+    pub fn block_on(
+        &self,
+        handle: impl Into<HandleUntyped>,
+    ) -> Result<HandleUntyped, AssetServerError> {
+        let mut handle = handle.into();
+        let path =
+            self.get_handle_path(handle.clone())
+                .ok_or(AssetServerError::AssetLoaderError(anyhow::anyhow!(
+                    "Handle does not have a corresponding Path"
+                )))?;
+        'block_on: loop {
```

Comment:
Why use a loop label here? This doesn't contain a inner loop. As a comment?

Reply:
Yeah its for readability but I can remove it